runners/java-fn-execution/build.gradle

```diff
@@ -58,3 +58,30 @@ task testDocker(type: Test) {
     includeCategories "org.apache.beam.runners.fnexecution.environment.testing.NeedsDocker"
   }
 }
+
+task runExpansionService (type: JavaExec) {
```

Comment:
+1 for relocating this, it is consistent with artifact staging

Reply:
IIUC java-fn-execution module is for runner specific code related to Fn API execution (job submission to job server, work execution) while core-construction-java contains code related to pipeline construction before submitting job to a runner, right ?

If so, I'm not sure if moving expansion service to the java-fn-execution module is the correct move. It might be weird (or impossible) to maintain expansion service in java-fn-execution while maintaining other SDK logic related to cross-language transform expansion in pipeline construction side.

If the need is to just package expansion service withe the job server offered by the runner, we should be able to do that without moving the expansion service, right ? (given that java-fn-execution depends on core-construction-java)

cc: @ihji @robertwb

Reply:
I agree with your arguments. I thought that `fn-execution` might be the better place because other services also live in this module. However, the `ExpansionService` is clearly related to the construction of the pipeline, not its execution. I'll prepare a PR to move it back.

Reply:
Thanks Max. Sorry about not commenting before the PR went in.